### PR TITLE
Change group by and moved to graph

### PR DESCRIPTION
**Pull request by TomRytt**
# Description
1. Moved the group by time/hour to the location of the graph it was manipulating
2. Changed the switch to a button
3. Improved the wording of the button and the title
4. Removed a "-" sign which was left alone in the top menu

## screenshots

Before:
image

After:
image

### CODE REVIEW

File: src/pages/dashboard/DashboardPage.tsx
Comment (shootermv) [inline src/pages/dashboard/DashboardPage.tsx:1]: Can you please remove the commented out code?
😀

Comment (TomRytt) [reply src/pages/dashboard/DashboardPage.tsx:1]: Sure, done :)

Comment (shootermv) [reply src/pages/dashboard/DashboardPage.tsx:1]: Good job!

**Comment by shootermv** [comment]
Can i ask you- replace button by two buttons
Where only one can be 'on' at the time
That way it will be clear what is the current mode
(I think it called radio buttons)

**Comment by TomRytt** [comment]
> Can i ask you- replace button by two buttons
> Where only one can be 'on' at the time
> תגוצה רגילה
> תצוגה לפי שעה
> That way it will be clear what is the current mode
> (I think it called radio buttons)
> Like [here](https://codesandbox.io/s/yvwzvm)

That is actually why I updated the title to show the current mode according to the state and then text on the button always shows the opposite. If you think it will still be better then sure, I'll do it this evening.

Also, I think that "תצוגה לפי יום" better reflects what it does than "תצוגה רגילה"

**Comment by TomRytt** [comment]
> you right, 
> it better to be: 
> לפי שעה & לפי יום
> but i still thinK it is better to have both buttons - so user will see all the possible options...
> image
> 

Great, I'll modify it this evening

**Comment by shootermv** [comment]
you right, 
it better to be: 
לפי שעה & לפי יום
but i still thing it is better to have both buttons - so user will see all the possible options...
image

**Review by NoamGaash — APPROVED** [review]
lgtm
thanks

**Comment by TomRytt** [comment]
> you right, it better to be: לפי שעה & לפי יום but i still thinK it is better to have both buttons - so user will see all the possible options... image

Changed to radio group buttons as you requested :)
image

**Review by shootermv — APPROVED** [review]
well done!

**Review by shootermv — CHANGES_REQUESTED** [review]
need to fix the playwright tests
(matter of changing the expected text)

**Comment by TomRytt** [comment]
> need to fix the playwright tests (matter of changing the expected text)

Done

**Review by shootermv — APPROVED** [review]

**Comment by shootermv** [comment]
Thank you for your contribution!
Hope you will continue to work on this project and make more good stuff

**Pull request merged**